docs/maps/leafycanyon.md

```diff
@@ -0,0 +1,40 @@
+<!-- replace _map_ with the actual map name -->
+<!-- change gamemode type for the Map data description  -->
+# Leafy Canyon
+
+***
+
+#### ![leafycanyonicon](../assets/maps/leafycanyon/leafycanyon-icon.jpg)
+
+# Overview
+***
+- **Introduced:** v1.7.0
+- **Description:** A woodland canyon.
```

Comment:
Please include a description of the vegetation in the description. Something like "A rocky and lush canyon with water falls"?